h264_encoder_core/src/h264_encoder.cpp

```diff
@@ -166,7 +179,7 @@ class H264EncoderImpl
     AVDictionary * opts = nullptr;
     if (codec_name.empty()) {
       codec = avcodec_find_encoder_by_name(kDefaultHardwareCodec);
-      if (nullptr == codec || !is_omx_available()) {
+      if (nullptr == codec || !is_omx_available() || !is_codec_available(codec)) {
```

Comment:
`is_omx_available()` is not longer required, right?

Reply:
Yep, removed and reran the above tests.

Reply:
Looks like this is causing unit tests to fail in bionic, not xenial though.